guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The implementation of the <s>above discussed </s><code>FullyConnectedNetwork</code> and <code>MLModel</code> architecture <s>could</s> <u>can </u>be better understood in one of the sample notebooks where energy generation from solar photovoltaic power plants using weather variables has been predicted using both <s>these </s>models and is available in this <a href="https://developers.arcgis.com/python/sample-notebooks/" target="_blank">repository</a>, along with other interesting notebooks based on the various modules from ArcGIS API for Python.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>